### ISSUE TITLE
Only first split is visible when time is false

### ISSUE THREAD

**Issue by Duncank** [antoniandre/vue-cal-v4#252]
With the following code, only the events and header-label for the first defined split are shown. 
Also [see this JSFiddle](https://jsfiddle.net/Duncankrebbers/zu5fhs48/16/). Only difference between the top and the bottom calendar is the `time: false` option.

```js
events: [
    {
        start: '2020-03-23 01:00',
        end: '2020-03-23 02:00',
        title: 'Title',
        split: 'user1'
    },
    {
        start: '2020-03-23 02:00',
        end: '2020-03-23 03:00',
        title: 'Title 2',
        split: 'user2'
    },
]
```

```js
options: {
    hideViewSelector: true,
    hideTitleBar: true,
    splitDays: [
        {
            id: 'user1',
            label: 'User 1'
        },
        {
            id: 'user2',
            label: 'User 2'
        }
    ],
    time: false
},
```

**Comment by antoniandre** [comment]
Hi @Duncank,
This is now fixed in version 3.0.2.

**Comment by Duncank** [comment]
Awesome, great work! I've updated the plugin and I can confirm it is working now.
